dev-itpro/developer/devenv-application-overview.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Overview of the Application"
+description: "This article provides an overview of the modules in the Application, and information about how you can use them."
+author: bholtorf
+ms.topic: overview
+ms.author: bholtorf
+ms.date: 01/08/2024
+ms.collection:
+  - bap-ai-copilot
+ms.reviewer: bholtorf
+---
+
+# Overview of the Application
+
+[!INCLUDE[azure-ad-to-microsoft-entra-id](~/../shared-content/shared/azure-ad-to-microsoft-entra-id.md)]
+
+The business central application code is being refactored from a single monolythic application into smaller applications (layers) which in turn are built from small modules which handle a single functionality or feature.
+
+The System Application layer contains modules that provide ERP agnostic functionality. These are mostly tecnical features which any application needs.
+The Business Foundation layer contains modules that provide ERP related functionality that applies to all or none of the application domains.
+
+As we continue to modernize the application code more modules and layers will be added. For an overview of system and application reference documentation, see [System and Base Application Reference for Dynamics 365 Business Central](/dynamics365/business-central/application).
+
+> [!Note]
+> The modules in the System Application and Business Foundation represent a significant change in what's happening under the hood in [!INCLUDE[d365fin_long_md](includes/d365fin_long_md.md)]. We are aware that the changes we have made will introduce breaking changes, so we have made a list of those that we know about, which includes suggestions for solutions. To view the breaking changes list, see [Breaking Changes](https://github.com/microsoft/ALAppExtensions/blob/master/BREAKINGCHANGES.md).<br><br>We will continue to enhance the System Application in future releases. If you find something you think we should add, visit our [Dynamics 365 Application Ideas](https://aka.ms/bcideas) page. If you want us to improve something, go to the [BCApps](https://github.com/microsoft/BCApps) repository and submit a pull request for it.
```

Comment:
The Breaking Changes page does not seem to be maintained anymore - which makes sense because real breaking changes are not allowed directly. The only exception was when the System App was introduced. Additionally, the link refers to the old repository ALAppExtensions. The current one is BCApps. Hence, I'm not sure what the benefit of mentioning it is. Is it for updating old On-Prem solutions to the current BC version? If so, then this could be mentioned.

At the very end, you ask the community to create a pull request to the BCApps repository - but that's too short. PRs without a previous and approved issue won't be processed. Please refer to Jesper, if phrasing it the current way was not already agreed with him.

Reply:
@NKarolak good points, how about this instead:

>[!Note]
> If you find something you think we should add, visit our [Dynamics 365 Application Ideas](https://aka.ms/bcideas) page. If you would like to contribute directly, you can do so in to the [BCApps](https://github.com/microsoft/BCApps) repository. Please review the contribution guidelines here: [Contribution Guidelines](https://github.com/microsoft/BCApps/blob/main/CONTRIBUTING.md)